### PR TITLE
PP-14 : Add project experiment page

### PR DESCRIPTION
**Pull request by sanjay-techuz**

### CODE REVIEW

**Comment by sanjay-techuz** [comment]
[Add project experiment page](https://trello.com/c/2e5EYu9h/14-add-project-experiment-page)

File: src/components/Table/ContentPlan/index.jsx [hunk 1]
```diff
@@ -73,7 +73,7 @@ function Row(props) {
           align="center"
           component="th"
           scope="row"
-          // onClick={() => navigate('/project-detail', { state: { row } })}
+          onClick={() => navigate('/content-plan-detail', { state: { row } })}
```
Comment (ashishtechuz) [inline src/components/Table/ContentPlan/index.jsx:76]: Do not pass an entire row as a state. always avoid these. Use query params to pass the ID and fetch the record based on that Id in detail page.

Comment (ashishtechuz) [reply src/components/Table/ContentPlan/index.jsx:76]: Reason behind this is that no one will be able to go to the project detail page with just a link. They must have to go to the list page and then only they'll be able to go to the detail page. Any shared link of project detail won't work.

File: src/components/Table/ProjectExperiment/DocumnetsAndLinks/index.jsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    </React.Fragment>
+  );
+}
+
+export default function Index(props) {
+  const { rows } = props;
+  const [page, setPage] = useState(0);
+  const [rowsPerPage, setRowsPerPage] = useState(10);
+
+  const handleChangePage = (event, newPage) => {
```
Comment (ashishtechuz) [inline src/components/Table/ProjectExperiment/DocumnetsAndLinks/index.jsx:33]:
```javascript
 const handleChangePage = (_event, newPage) => {
```

**Review by ashishtechuz — CHANGES_REQUESTED** [review]

**Review by ketanp-techuz — DISMISSED** [review]

**Review by ashishtechuz — APPROVED** [review]

**Pull request merged**